interpreter/exec/eval.ml

```diff
@@ -42,6 +42,11 @@ let numeric_error at = function
   | exn -> raise exn
 
 
+let value_of_index it x =
```

Comment:
Perhaps move this to value.ml

Reply:
I don't see that file.. are you suggesting a new file with this one function?

Reply:
Sorry, syntax/values.ml